### ISSUE TITLE
jwt.isExpired throws a TypeError after being verified

### ISSUE THREAD

**Issue by anthonyalmarza** [okta/okta-jwt-verifier-js#7]
## I'm submitting this issue for the package(s):

- [X] jwt-verifier
- [ ] oidc-middleware

## I'm submitting a:

- [X] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior

After successfully calling `oktaVerifier.verifyAccessToken` to generate an object with the `OktaJwtVerifier.Jwt` interface calling `isExpired` on the jwt throws a `TypeError`.

```
jwt.isExpired() // throws TypeError: Cannot read property 'exp' of undefined
```

## Expected behavior

```
jwt.isExpired() // should return a boolean
```

## Minimal reproduction of the problem with instructions

On `nJwt` the `JWT` prototype expects there to be a `body` attribute on the object [ref](https://github.com/jwtk/njwt/blob/master/index.js#L265-L267) whereas on this repo it's being deleted [ref](https://github.com/okta/okta-jwt-verifier-js/blob/master/lib.js#L208).

## Extra information about the use case/user story you are trying to implement

It'd be nice to be able to follow the interfaces described in `lib.d.ts`

## Environment

- Package Version: 2.3.0
- OS: Ubuntu
- Node version (`node -v`): 12

**Comment by denysoblohin-okta** [comment]
Thanks for submitting this issue and finding the cause.
Internal ref: OKTA-454520

**Comment by riasarkar93** [comment]
also having this issue -- any update @denysoblohin-okta ?

**Comment by oleksandrpravosudko-okta** [comment]
Fixed in 2.6.0
